specification/containerservices/resource-manager/Microsoft.ContainerService/2017-08-31/examples/ManagedClustersAccessProfilesGet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2017-08-31",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "accessProfileRoleName": "clusterUser"
```

Comment:
There is a model validation requires a `roleName` parameter, but you have this. Can you please update with the correct one